x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/BooleanBlock.java

```diff
@@ -157,12 +157,24 @@ static int hash(BooleanBlock block) {
         return result;
     }
 
+    /** Returns a builder using the {@link BlockFactory#getNonBreakingInstance block factory}. */
+    // Eventually, this should use the GLOBAL breaking instance
```

Comment:
I guess we'd remove this entirely when we remove the static access, right?

Reply:
ah yes, I'll update the comment

Reply:
:+1:

Reply:
Done